circle.yml

```diff
@@ -0,0 +1,4 @@
+test:
+  override:
+    - npm test
+    - bin/lint
```

Comment:
yea, policial runs eslint under the hood, but this is easier for us. I made bin/lint because its a pita to run the command all the time